packages/alfa-style/test/property/scale.spec.tsx

```diff
@@ -0,0 +1,142 @@
+import { h } from "@siteimprove/alfa-dom";
+import { test } from "@siteimprove/alfa-test";
+
+import { Device } from "@siteimprove/alfa-device";
+
+import { Style } from "../../dist/style.js";
+
+const device = Device.standard();
+
+test("initial value is `none`", (t) => {
+  const element = <div>Hello</div>;
+
+  const style = Style.from(element, device);
+
+  t.deepEqual(style.computed("scale").toJSON(), {
+    value: {
+      type: "keyword",
+      value: "none",
+    },
+    source: null,
+  });
+});
+
+test("property is not inherited", (t) => {
+  const element = <div>Hello</div>;
+
+  h.document(
+    [<div class="container">{element}</div>],
+    [h.sheet([h.rule.style(".container", [h.declaration("scale", "2")])])],
```

Comment:
**(optional)**
```suggestion
    [h.sheet([h.rule.style(".container", { "scale", "2" })])],
```
When the declaration is not needed, we can use a name/value record which is arguably easier to read for humans.